# <center>Набор данных оценки недвижимости</center>
#### <center>Датасет можно скачать здесь: https://archive.ics.uci.edu/ml/machine-learning-databases/00477/</center>

In [8]:
import pandas as pd

real_estate_valuation_data=pd.read_excel('Real estate valuation data set.xlsx', index_col=0)

In [11]:
real_estate_valuation_data.head()

X1 transaction date  X2 house age  X3 distance to the nearest MRT station  \
No                                                                              
1           2012.916667          32.0                                84.87882   
2           2012.916667          19.5                               306.59470   
3           2013.583333          13.3                               561.98450   
4           2013.500000          13.3                               561.98450   
5           2012.833333           5.0                               390.56840   

    X4 number of convenience stores  X5 latitude  X6 longitude  \
No                                                               
1                                10     24.98298     121.54024   
2                                 9     24.98034     121.53951   
3                                 5     24.98746     121.54391   
4                                 5     24.98746     121.54391   
5                                 5     24.97937     121.54245   

    Y house price of unit area  
No                              
1                         37.9  
2                         42.2  
3                         47.3  
4                         54.8  
5                         43.1

### Общая информация: рыночные исторические данные об оценке недвижимости собранные в Синдиане, Нью-Тайбэй, Тайвань
### Информация об атрибутах: 
    Признаки:
        X1 the transaction date - дата транзакции (например: 2013.250=2013 Март, 2013.500=2013 Июнь и тд.)
        X2 house age - возраст дома (в годах)
        X3 distance to the nearest MRT station - расстояние до ближайшей станции метро (в метрах)
        X4 number of convenience stores - количество круглосуточных магазинов в жилом круге пешком (целое число)
        X5 latitude - географическая координата, широта. (в градусах)
        X6 longitude - географическая координата, долгота. (в градусах)
    Целевая переменная: 
        Y house price of unit area - цена дома за единицу площади (10000 новых тайваньских долларов / пинг, где пинг является местной единицей, 1 пинг = 3,3 метра в квадрате)
### Характеристики набора данных: многомерные
### Характеристики атрибутов: целые и вещественные числа
### Задача: регрессии
### Количество наблюдений: 414
### Количество признаков: 7
### Пропущенные значения: отсутствуют
### Область: бизнес
### Дата сбора выборки: 2018-08-18
### Количество посещений: 7439
### Источник: Prof. I-Cheng Yeh, Строительный факультет, Университет Тамканга, Тайвань 